pbsmetrics/prometheus/prometheus.go

```diff
@@ -57,16 +56,12 @@ func NewMetrics(cfg config.PrometheusMetrics) *Metrics {
 		[]string{"ErrorType"},
 	)
 	metrics.Registry.MustRegister(metrics.connError)
-	metrics.imps = newCounter(cfg, "imps_requested_total",
-		"Total number of impressions requested through PBS.",
-		standardLabelNames,
-	)
-	metrics.Registry.MustRegister(metrics.imps)
-	metrics.impTypes = newCounter(cfg, "imps_types_total",
-		"Total number of impression types requested through PBS.",
+	metrics.imps = newCounter(cfg, "imps_requested_total_by_type",
```

Comment:
Can shorten the name to `imps_requested_total` or even `imps_requested` ... the "by type" is implied by the labels available.